### PR TITLE
Add charts for scores

### PR DESCRIPTION
**Pull request by yujinjcho**

### CODE REVIEW

**Comment by yujinjcho** [comment]
lol check this out

#### Section I had for cv set
cv_set

#### Section I had for test set
test_set

**Comment by yujinjcho** [comment]
So I used another group of the part_files and looks like lambda 10 is working the best
### cv_set_ac
cv_set_ac

### test_set_ad
test_set_ad

**Comment by yujinjcho** [comment]
This is pretty kewl. susan-granger gives fresh rating ~57% of time. Model ~80%+ correct. Also, some pretty high lambdas

### susan-granger cv set
susan-granger-cv

### susan-granger test set
susan-granger-test

**Comment by achiu8** [comment]
Interesting.. so does this mean it's just severely overfitting the training set? Because of how many features we have?

**Comment by yujinjcho** [comment]
I think so
